app/views/campaigns/edit.html.haml

```diff
@@ -139,7 +140,15 @@
           %p.rails_editable-content
             = preserve @campaign.template_description
           = text_area(:campaign, :template_description, class: 'rails_editable-input', maxlength: 65535)
-
+      - if (current_user&.admin?)
```

Comment:
This change is good, I think, and we can keep it independent of the React conversion of the overview tab.

Reply:
What do you mean by "we can keep it independent of the React conversion of the overview tab"? Does that mean I can not remove the code line number 41 to 49 in the overview.html.haml  that relates to the Disable/ Enable account request button?

Reply:
Yes, exactly.

Reply:
I added that code to overview.html.haml file. So I can keep those lines that belong
 to Disable/Enable account request button in the edit.html.haml file?